### ISSUE TITLE
Difficulty seeing training and validation accuracy in graph

### ISSUE THREAD

**Issue by rm51** [udacity/sdc-issue-reports#120]

**Comment by rm51** [comment]
Perhaps the two colors can be changed to something else in the TensorFlow lab. They overlap so it's difficult to see that they're exactly the same. 

For instance 

acc_plot.plot(batches, train_acc_batch, 'r', label='Training Accuracy')
acc_plot.plot(batches, valid_acc_batch, 'b', label='Validation Accuracy')

**Comment by rm51** [comment]
Perhaps it could be changed to circles and xs.

**Comment by Brok-Bucholtz** [comment]
@rm51,

Validation accuracy is using X's instead of a line.  This should make the accuracy easier to read.
